utils.py

```diff
@@ -177,3 +177,21 @@ def send_flow_mods_event(
                 "force": force,
             },
         )
+
+
+def prepare_delete_flow(evc_flows: dict[str, list[dict]]):
+    """Create flow mods suited for flow deletion."""
+    dpid_flows: dict[str, list[dict]] = {}
+
+    if not evc_flows:
+        return dpid_flows
+
+    for dpid, flows in evc_flows.items():
+        dpid_flows.setdefault(dpid, [])
+        for flow in flows:
+            dpid_flows[dpid].append({
+                "cookie": flow["cookie"],
+                "match": flow["match"],
+                "cookie_mask": int(0xffffffffffffffff)
```

Comment:
The `cookie_mask` fix itself looks good, and it should work since `table_id` defaults to 0. 

Since this is a specific match deletion for a given flow and not all flows of a dpid, parametrizing the `table_id` value should be the way to go. But I just realized that we also have this problem for other deletions too on `mef_eline`, none of them are parametrizing `table_id` and a flow mod deletion message serialized defaults to 0.

Can you confirm that for mef_eline if using a different table indeed it would leave flows behind? No need to fix that part on this PR, let's just confirm the issue and map it, and then address/prioritize later.

Reply:
They are deleted. But the action is only taken in the database then consistency check from `flow_manager` checks database and then deletes the correct flows in the topology.

Reply:
Cool. Great to have this confirmation, yes, the behavior isn't great. So, for cases where mef_eline isn't using table 0 this becomes a bit cumbersome. I mapped a new issue https://github.com/kytos-ng/flow_manager/issues/195 not set for version `2024.1` since in prod we know that mef_eline is only at table 0 for now, let's see how that will unfold and we'll reasess later.